Common/Deployment/DeploymentLib.ps1

```diff
@@ -802,7 +805,7 @@ function InitializeEnvironment()
             UpdateEnvSetting "SubscriptionId" $global:SubscriptionId
         }
     }
-    Select-AzureSubscription -SubscriptionId $global:SubscriptionId
+    
```

Comment:
Looks like it does work without this line, and fails if you haven't run add-azureaccount.

Closed as is.